src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-add-content.js

```diff
@@ -59,7 +59,14 @@
 
         if (elementPath && elementPath.block) {
             const elements = elementPath.elements;
-            const insertIndex = !elementPath.contains(isCustomTag, true) ? elements.length - 2 : 0;
+            const insideTableCell = elementPath.blockLimit && elementPath.blockLimit.getName() == 'td';
+            const insideCustomTag = elementPath.contains(isCustomTag, true);
+            const isCurrentElementList = element.getName() == 'ul' || element.getName() == 'ol';
+
+            let insertIndex = insideTableCell || insideCustomTag ? 0 : elements.length - 2;
+            if (elementPath.lastElement.getName() == 'li') {
```

Comment:
And here I have a question why you have to check if this is a list item and increase the index

Reply:
@dew326 Answering your question:
If you are trying to insert any content (embed/image/etc) into the list item, it will be added after the list element, but not the list item element. And it fixes list item related issues previously submitted by @barbaragr 
I guess this behavior will be changed in the future when it will be allowed to insert embed/images/etc inside list items.